opentelemetry-sdk/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## vNext
 
 - *Breaking* Make `force_flush()` in `PushMetricExporter` synchronous
+- Moved `ExportError` trait from `opentelemetry::trace::ExportError` to `opentelemetry_sdk::export::ExportError`
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/pull/2654/files will make further changes to these, likely getting rid of these completely.

I can probably fix changelog at that time to reflect the net effect of the change.